sys/stm32/machdep.c

```diff
@@ -319,6 +319,9 @@ cpuidentify()
         case 0x2000:
             printf("A");
             break;
+	case 0x1001:
+            printf("VG");
```

Comment:
This is incorrect. DBGMCU_IDCODE REV_ID of 0x1001 is Z, not VG.
Check section 36.6.1 of the reference manual RM0090 for all the possible revisions of REV_ID.